pkg/monitortests/node/watchnodes/node_test.go

```diff
@@ -55,3 +57,283 @@ func TestNodeRoles(t *testing.T) {
 		}
 	}
 }
+
+func TestReportUnexpectedNodeDownFailures(t *testing.T) {
+	var testCases = []struct {
+		name             string
+		rawIntervals     monitorapi.Intervals
+		unexpectedReason monitorapi.IntervalReason
+		expected         []string
+	}{
+		{
+			name: "node unexpected ready reason with no deleted machines",
+			rawIntervals: monitorapi.Intervals{
+				{
+					Condition: monitorapi.Condition{
+						Level: monitorapi.Error,
+						Locator: monitorapi.Locator{
+							Type: monitorapi.LocatorTypeNode,
+							Keys: map[monitorapi.LocatorKey]string{
+								"node": "node1",
+							},
+						},
+						Message: monitorapi.Message{
+							Reason:       monitorapi.NodeUnexpectedReadyReason,
+							HumanMessage: "unexpected node not ready",
+							Annotations: map[monitorapi.AnnotationKey]string{
+								monitorapi.AnnotationReason: "UnexpectedNotReady",
+							},
+						},
+					},
+					From: utility.SystemdJournalLogTime("Nov 11 19:46:00"),
+					To:   utility.SystemdJournalLogTime("Nov 11 19:46:00"),
+				},
+			},
+			expected:         []string{"node/node1 - reason/UnexpectedNotReady unexpected node not ready at from: 2024-11-11 19:46:00 +0000 UTC - to: 2024-11-11 19:46:00 +0000 UTC"},
+			unexpectedReason: monitorapi.NodeUnexpectedReadyReason,
+		},
+		{
+			// OCPBUG-44244: if machine monitor does not have node name than we will get no failures
```

Comment:
This test looks to expect a failure. Just a comment that needs updating?

Reply:
yea. updated.